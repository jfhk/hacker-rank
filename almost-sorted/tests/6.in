10
2 4 6 8 12 10 14 16 18 20
