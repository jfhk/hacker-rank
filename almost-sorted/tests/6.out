yes
swap 5 6
